addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6689,23 +6773,24 @@ registry.ImageTools = ImageHandlerOption.extend({
                         img.classList.remove("o_animate_on_hover");
                     }
                     if (!this._isAnimatedShape()) {
-                        delete img.dataset.shapeAnimationSpeed;
+                        delete this.imageOptions.shapeAnimationSpeed;
                     }
                 }
+                await this._applyShapeAndColors(saveData);
```

Comment:
This has to be at the end of the `imageOptions` modification in order to correctly update the `image.options` registry

Reply:
Feels like a comment that could be in the code directly :+1: